data-structures/stacks-1.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { Stack } from './lib/stack';
+
+export function getTop<T>(stack: Stack<T>): T | undefined {
+  const result = stack.peek();
+  if (!stack) {
+    return undefined;
+  }
+  return result;
+}
+
+export function addToTop<T>(stack: Stack<T>, value: T): void {
+  if (!stack || !value) {
```

Comment:
It's good to check `value` here, since you cannot push `undefined` onto a stack. However, `stack.push` makes the same check so it's not needed here.